qa/L0_shared_memory/shared_memory_test.py

```diff
@@ -323,6 +324,29 @@ def test_unregisterall(self):
             self.assertTrue(len(status_after.regions) == 0)
         self._cleanup_server(shm_handles)
 
+    def test_infer_offset_out_of_bound(self):
+        # Shared memory offset outside output region - Throws error
+        error_msg = []
+        shm_handles = self._configure_sever()
+        if _protocol == "http":
+            # -0x20 in int64_t
+            offset = 18446744073709551584
```

Comment:
what's the significance of this value? Do we have a known bounds in this test, and can instead just call "max_bounds+1" or "min_bounds-1" or something?

Reply:
I think putting this number in some format like `2**64` or `2**64 - 31` would be more clear in what you're trying to express here

Reply:
Sure. [Improve comments](https://github.com/triton-inference-server/server/pull/6914/commits/2bd4cd8ec333c064b13533f54fb8f666396ae626)

Reply:
Thanks for improving the comments, for the actual offset itself, I meant changing the number directly to be clear:
```suggestion
            offset = 2**64 - 32
```

Reply:
[Improve comment and offset](https://github.com/triton-inference-server/server/pull/6914/commits/bf5a8dc4bc84acb135c49a6bf9a6e73f89075aee)